terraform/a2ha-terraform/reference_architectures/aws/main.tf

```diff
@@ -1,6 +1,6 @@
 provider "aws" {
   region  = var.aws_region
-  profile = var.aws_profile
+  profile = length(var.aws_profile) > 0 ? var.aws_profile : ""
```

Comment:
Is there a reason for doing this test? If the length is <= 0 then it should already be an empty string or a null value in which case it should grab the default value if any is defined. If you are trying to ensure that this is an empty string instead of null then it should have the default value set in the variables file.

Reply:
Made its default value as empty and if there is any it will pick it up